hack/local-up-volcano.sh

```diff
@@ -41,7 +39,10 @@ function prepare {
 
 function install-volcano {
   # TODO: add a graceful way waiting for all crd ready
-  kubectl apply -f ${RELEASE_FOLDER}/${YAML_FILENAME}
+  kubectl create namespace volcano-system 
+  helm install volcano ${VK_ROOT}/installer/helm/chart/volcano --namespace volcano-system \
```

Comment:
[#2936](https://github.com/volcano-sh/volcano/pull/2936) has added depend information to Volcano's helm package. Before installing here, do you need to execute `helm dependency build` first?

Reply:
done